### ISSUE TITLE
"add to library" lacks of discoverability

### ISSUE THREAD

**Issue by mjourdan** [haecker-felix/Gradio#321]
Use case: while listening to a specific radio station, I want to add other radio stations to my library without interrupting the radio currently playing.

## Expected Behavior

When browsing stations, there is an obvious way to add a single station to the library

## Current Behavior

The current process is a bit hard to discover:

- left-click on a radio station plays it, but doesn't give a clue on how to add it to the library
- the user has to switch to selection mode, select the radio station, then click the "add to library" button (selection mode may be ok for dealing with multiple items, not really otherwise)
- controls are split across the action bar (bottom) and details pane (right) 

## Proposal

Here are some ideas to get things more straightforward:

- automatically reveal the details pane on left click
- gather all controls in the details pane
- change the station playing only when clicking on the "play" button in the details pane

![image](https://github.com/mjourdan/gradio-design/blob/master/gradio-add-to-library.png?raw=true)

I would be glad to read your thoughts on this!

**Comment by sispus** [comment]
hi,
I think right clicking on a station already does what you suggest. Wrong?

**Comment by mjourdan** [comment]
Nope, right clicking on a station enters [selection mode](https://developer.gnome.org/hig/stable/selection-mode.html.en), which I think should be avoided for manipulating a single element.
